client/protocols/backup.py

```diff
@@ -176,4 +188,21 @@ def send_messages(self, access_token, messages, timeout):
         response = urllib.request.urlopen(request, serialized_data, timeout)
         return json.loads(response.read().decode('utf-8'))
 
+    @staticmethod
+    def cp(msg):
+        try:
+            # windows
+            if os.name == 'nt':
+                res = os.system('echo ' + msg.strip() + '| clip')
```

Comment:
I tried to use `subprocess` to replace `os.system`, but it's really hard on Windows. Another thing we can do is to use `pyperclip`, but it's a third party library.